### ISSUE TITLE
systemd complains about using /var/run dir for PID files

### ISSUE THREAD

**Issue by sharkcz** [ibm-s390-tools/s390-tools#52]
Some of the unit files in systemd directory use /var/run directory for PIDFiles and systemd complains with eg.
````
/usr/lib/systemd/system/cpacfstatsd.service:8: PIDFile= references path below legacy directory /var/run/,
updating /var/run/cpacfstatsd.pid → /run/cpacfstatsd.pid; please update the unit file accordingly.
````
Can we switch to /run? Or use a substitution from a Makefile variable?

**Comment by hoeppnerj** [comment]
I found a few other occurrences where we use /var/run and I think it's totally fine to change these to /run.
Please send a PR with the changes if you want.

**Comment by sharkcz** [comment]
fixed via commit 847f16f632d88c539b84292e8297bb4932562d6e
